shepherd_bms/src/main.cpp

```diff
@@ -140,7 +133,7 @@ void shepherdMain()
 	if (currTime > lastStatMsg + 500) {
 		lastStatMsg = currTime;
 		Serial.print("Current: ");
-		Serial.println(current);
+		Serial.println(accData->packCurrent);
 		Serial.print("Min, Max, Avg Temps: ");
```

Comment:
might be getting redundant/overkill at this point but once again for pure readability, might be worth considering packaging all this data printing into one function